pkg/graveler/graveler.go

```diff
@@ -1571,6 +1571,29 @@ func (g *KVGraveler) listStagingArea(ctx context.Context, b *Branch) (ValueItera
 	return NewCombinedIterator(itrs...), nil
 }
 
+func (g *KVGraveler) listSealedTokens(ctx context.Context, b *Branch) (ValueIterator, error) {
+	var itrs []ValueIterator
```

Comment:
can allocate the slice size based on the length of sealed tokens (at least the capacity)

Reply:
it's not working if the list is empty

Reply:
done